docs/api.rst

```diff
@@ -703,6 +703,60 @@ Getting or modifying settings
    :statuscode 409: No user is logged in or tried to set eth rpc endpoint that could not be reached.
    :statuscode 500: Internal rotki error
 
+Getting backend server arguments
```

Comment:
I would like to switch towards seeing rotki as a collection of local daemons that will run together, so don't want to name something as server. Say either `backend` or maybe at some point we can say `core` 

```suggestion
Getting backend arguments
```